core/src/main/java/io/spine/core/CommandMixin.java

```diff
@@ -88,4 +91,20 @@ default boolean isScheduled() {
         }
         return false;
     }
+
+    @Override
+    default Object readValue(Descriptors.FieldDescriptor field) {
+        switch (field.getIndex()) {
+            case 0:
```

Comment:
Can we use named constants for field numbers instead?

Reply:
We would have to depend on the descendants (i.e. on the classes, which are marked by this mixin interface), as the `...OrBuilder` has no such notion. So, unfortunately, no.